automatic/gimp/gimp.nuspec

```diff
@@ -16,9 +17,8 @@ GIMP has many capabilities. It can be used as a simple paint program, an expert
 
 GIMP is expandable and extensible. It is designed to be augmented with plug-ins and extensions to do just about anything. The advanced scripting interface allows everything from the simplest task to the most complex image manipulation procedures to be easily scripted.
     </description>
-    <summary>GIMP</summary>
-    <releaseNotes></releaseNotes>
-    <copyright>GPL</copyright>
+    <releaseNotes>http://www.gimp.org/release-notes/</releaseNotes>
```

Comment:
@dtgm The release notes section is capable of doing markdown.  Could you make this URL a markdown URL so that it is clickable?